proto/umee/leverage/v1/tx.proto

```diff
@@ -22,6 +22,10 @@ service Msg {
   // exchange for burning uTokens.
   rpc Withdraw(MsgWithdraw) returns (MsgWithdrawResponse);
 
+  // WithdrawMaximum moves previously supplied tokens from the module back to the user balance in
+  // exchange for burning uTokens. It automatically calculates the maximum valid amount to withdraw.
+  rpc WithdrawMaximum(MsgWithdrawMaximum) returns (MsgWithdrawMaximumResponse);
```

Comment:
```suggestion
  rpc MaxWithdraw(MsgMaxWithdraw) returns (MsgMaxWithdrawResponse);
```

Reply:
Let's be consistent with Query name

Reply:
These message names are designed to make sense verbally, hence the order

`MsgWithdrawMaximum`: "Withdraw the maximum amount"
vs
`QueryMaxWithdraw`: "Query the maximum an account can withdraw"

Reply:
I still think it would make more sense to call it ` MsgMaxWithdraw`, rather than adding `Amount` to the original name.
At least, let's use `Max` rather than `Maximum`

Reply:
Renamed to `MsgMaxWithdraw`